TodoApp.IntegrationTesting/Fixtures/UserFixture.cs

```diff
@@ -0,0 +1,31 @@
+﻿
+
+using MongoDB.Driver;
+
+namespace TodoApp.IntegrationTesting.Fixtures
+{
+    public static class UserFixture
+    {
+        public static User GetUser() =>
+            new()
+            {
+                CreatedDateTime = DateTime.Now,
+                FirstName = "FirstName",
+                LastName = "LastName",
+                Role = "User"
```

Comment:
Avoid static data for test data, you can use some library like autofixture,bugous,etc to generate data